tests/performance_tests/performance_test_basic.py

```diff
@@ -445,11 +460,14 @@ def main():
     extraNodeosHttpPluginArgs = PerformanceBasicTest.ClusterConfig.ExtraNodeosArgs.ExtraNodeosHttpPluginArgs(httpMaxResponseTimeMs=args.http_max_response_time_ms)
     extraNodeosArgs = PerformanceBasicTest.ClusterConfig.ExtraNodeosArgs(chainPluginArgs=extraNodeosChainPluginArgs, httpPluginArgs=extraNodeosHttpPluginArgs, producerPluginArgs=extraNodeosProducerPluginArgs)
     testClusterConfig = PerformanceBasicTest.ClusterConfig(pnodes=args.p, totalNodes=args.n, topo=args.s, genesisPath=args.genesis, prodsEnableTraceApi=args.prods_enable_trace_api, extraNodeosArgs=extraNodeosArgs)
+    oldNodeos = False
 
+    if Utils.getNodeosVersion().split('.')[0] == "v2":
+        oldNodeos = True
```

Comment:
Potential one-liner? `oldNodeos = Utils.getNodeosVersion().split('.')[0] == "v2"`

If you push this up a line or two you can simply set `oldNodeos` in the constructor for `testClusterConfig`.